Calculator/Calculator/Model/Operator.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  Operator.swift
+//  Calculator
+//
+//  Created by 무리 on 2023/01/27.
+//
+
+enum Operator: Character, CaseIterable {
+    case add = "+"
+    case substract = "-"
+    case divide = "/"
+    case multiply = "*"
+    
+    func calculate(lhs: Double, rhs: Double) -> Double {
+        var result: Double
+        switch self {
+        case .add: result = add(lhs: lhs, rhs: rhs)
+            return result
+        case .substract: result = substract(lhs: lhs, rhs: rhs)
+            return result
+        case .divide: result = divide(lhs: lhs, rhs: rhs)
+            return result
+        case .multiply: result = multiply(lhs: lhs, rhs: rhs)
+            return result
+        }
+    }
```

Comment:
외부에서 사용될 calculate 메서드도 잘 정의 해주셨네요 👍

Reply:
```suggestion
    func calculate(lhs: Double, rhs: Double) -> Double {
        var result: Double
        switch self {
        case .add:
            result = add(lhs: lhs, rhs: rhs)
            return result
        case .substract: 
            result = substract(lhs: lhs, rhs: rhs)
            return result
        case .divide:
            result = divide(lhs: lhs, rhs: rhs)
            return result
        case .multiply: 
            result = multiply(lhs: lhs, rhs: rhs)
            return result
        }
    }
```

이렇게 개행하는 것이 더 자연스러울 것 같아요 🧐